### DISCUSSION TITLE
I dont know what had happened

### DISCUSSION THREAD

**Discussion by oishi81** [JanDeDobbeleer/oh-my-posh#3070] [Troubleshoot]
When I use notepad.exe $PROFILE and use the setting "oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\montys.omp.json" | Invoke-Expression". Save and reopen powershell, A bug or not happened.
I dont know why and wonder if anyone can help me. 
![image](https://user-images.githubusercontent.com/65519561/201513755-8e638594-505e-4105-aa4f-c369bc4b1aa8.png)

**Comment by oishi81** [comment]
I searched my pc and found there is oh-my-posh3(version 6.24.x) in scoop. Now I deleted oh-my-posh3, then it worked well.

**Reply by JanDeDobbeleer** [reply]
Yes, I was about to reply you had an old version that's causing this.
